docs/source/submission_strategies.rst

```diff
@@ -0,0 +1,145 @@
+.. _submission_strategies:
+
+*************************
+Job Submission Strategies
+*************************
+This page provides examples on how to optimize JADE for different types of use
+cases.
+
+Independent, short, multi-core jobs
+===================================
+
+Constraints:
+
+- Each job process consumes all cores of the compute node.
+- Each job takes no more than 30 minutes.
+- There are 1000 jobs.
+- There are no job dependencies.
+- Nodes on the short queue (4-hour max walltime) have a short acquisition time.
+- Nodes on the standard queue have a long acquistion time.
+
+Strategy:
+
+- Use the short queue (``partition = "short"`` in ``hpc_config.toml``).
+- Limit the number of parallel processes on each node (default is num CPUs).
+- Set the per-node batch size to the number of jobs that can complete within
+  the max walltime.
+- Try to use as many nodes as possible in parallel.
+
+Command:
+
+.. code-block:: bash
+
+    jade submit-jobs --num-processes=1 --per-node-batch-size=8 --max-nodes=125 config.json
```

Comment:
nit: add a `$` for commands (same below)? `$jade submit-jobs` can indicate this is a command, easier to tell from outputs (if any).

Reply:
Fixed